benchmarks/README.md

```diff
@@ -1,4 +1,19 @@
-These are the benchmarks used in testing the different WebSocket server implementations. I use WebSocket++ as clients, since that is the fastest client library that I know of, and since you can run multiple instances of the client, it shouldn't really matter what client you use as long as the server is being stressed fully.
+# uWebSockets Benchmarks
+
+## Building Instructions
+
+First, install dependencies:
+
+* `bench1` depends only uWebSockets.
```

Comment:
Oh, sorry for the typo and misunderstanding. I saw your fix at efb98947eb65cb61d0e3363d0ae9e03f54254729 .